.github/workflows/Weekly_Perf_Env_Installer_CI.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           END
       - name: ▶ OCP assisted installer
         env:
-          INSTALL_OCP_VERSION: "latest-4.12.0-rc"
+          INSTALL_OCP_VERSION: "latest-4.12"
```

Comment:
This belongs in a separate PR.

Reply:
Change is already in main.  This should not have been committed here, but it should not cause a problem.